src/MarkovKernels.jl

```diff
@@ -44,8 +44,8 @@ include("utils.jl") # helper functions
 include("affinemap.jl") # define affine maps to use as conditional means
 export AbstractAffineMap, AffineMap, LinearMap, AffineCorrector, slope, intercept, compose
 
-include("covariance_parameter.jl")
-export CovarianceParameter, lsqrt, stein, schur_reduce
+include("covariance_parameters/covariance_parameters.jl")
+export CovarianceParameter, SelfAdjoint, selfadjoint, rsqrt, lsqrt, stein, schur_reduce
```

Comment:
Should all of these types actually be exported to users? Do I need to use `CovarianceParameter` and `SelfAdjoint` myself?

Reply:
```CovarianceParameter``` / ```SelfAdjoint``` are not necessary.